packages/ui/src/components/Navigation/Navigation.stories.tsx

```diff
@@ -0,0 +1,880 @@
+/* eslint-disable @typescript-eslint/naming-convention */
+
+import type { Category, NewsroomLanguageSettings } from '@prezly/sdk';
+import type { Meta, StoryFn } from '@storybook/react';
+
+import { Navigation } from './Navigation';
+
+export default {
+    title: 'Components/Navigation',
+    component: Navigation,
+    parameters: { layout: 'fullscreen' },
+} as Meta<typeof Navigation>;
+
+const NavigationTemplate: StoryFn<typeof Navigation> = (args) => <Navigation {...args} />;
+
+export const Default = NavigationTemplate.bind({});
```

Comment:
Please add more stories with different variations.
There are two layouts, so that's two stories already.
Then it's good to show how would navigation look without categories, without an external link, without languages, without site logo, etc.